### ISSUE TITLE
Child - Admin | Sync Logs

### ISSUE THREAD

**Issue by aakashpatel660** [microsoft/powerapps-tools#477]
**Describe the bug**
When I go and look into Office 365 compliance & security portal I see a lot of PowerApps Launch audit logs but when my flow runs it does not look for that and does not collect those data. As you can on both screenshot 1 is for the flow run history and another is from Office 365 Compliance & security logs. 

**Component (please tell us which flow or app you are experiencing issues with):**
 - Solution - Audit Logs
 - Version - 1.0.0.17
 - App or Flow - Child - Admin | Sync Logs

**To Reproduce**

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/59940632/93639651-5cdcbe00-f9c7-11ea-917f-e7ecae678452.png)

![image](https://user-images.githubusercontent.com/59940632/93639929-be9d2800-f9c7-11ea-8561-5502121e6c53.png)

**Additional context**

**Comment by JeneferM-MSFT** [comment]
Hi. Thanks for using CoE. The screenshot you captured is not for a PowerApps Launch, so we would expect the behavior of no-oping there. 

You can scroll through the iterations with this UX and find a Launch case if you'd like to see if its working there. 
![image](https://user-images.githubusercontent.com/62659043/93791945-b58ba100-fbe9-11ea-8dc3-b2b0e985be3c.png)

It's likely easier to first check and see if the data come through to the entity:
![image](https://user-images.githubusercontent.com/62659043/93792095-ecfa4d80-fbe9-11ea-81df-cd8fc6787877.png)

![image](https://user-images.githubusercontent.com/62659043/93792202-0d2a0c80-fbea-11ea-8203-97fd2a30abb8.png)

**Comment by aakashpatel660** [comment]
Hello, 

I am trying to import a new version that was published today, and I am receiving an error. 

[Center of Excellence - Core Components_import.txt](https://github.com/microsoft/powerapps-tools/files/5257686/Center.of.Excellence.-.Core.Components_import.txt)

![image](https://user-images.githubusercontent.com/59940632/93811320-f1d8f480-fc1d-11ea-8e12-12a573b202ec.png)

This is the step I followed to import the new version. 

![image](https://user-images.githubusercontent.com/59940632/93811297-e5ed3280-fc1d-11ea-8b47-5e1bcbc402b2.png)

**Comment by manuelap-msft** [comment]
Hello,

could you please try and re-download the file and import again? We had a glitch in the uploaded file for a few hours which could lead to this issue, but have fixed it yesterday evening. 

Thank you
Manuela

**Comment by aakashpatel660** [comment]
Hello, 

I was able to import a new version of CoE into my environment but I ran into 1 issue. When I was importing the Audit Logs solution it was asking me for a connection. This connection has not been created and I am guessing it is a new connection. Until I do not import the solution it will not let me create a connection because the custom connector does not exist. Please advise

![image](https://user-images.githubusercontent.com/59940632/93889201-ebe32200-fcb6-11ea-88c5-22dfadcf87f3.png)

**Comment by manuelap-msft** [comment]
Hello,

have you also updated the new Core solution to the latest version?

If so, 
Go to make.powerapps.com > Data > Custom Connectors. The Office 365 Management API custom connector will be listed here,it has been imported with the Core components solution.

You will have to have configured the custom connector prior to this following steps 5 onwards here (although you might have already done that)
**https://docs.microsoft.com/en-us/power-platform/guidance/coe/setup-auditlog#set-up-the-custom-connector**

**Comment by aakashpatel660** [comment]
Hello 

I have already created a connection. As you can see below I have created a connection again. 
![image](https://user-images.githubusercontent.com/59940632/93907880-efcd6f00-fccb-11ea-9bf7-94e903b6e597.png)

After creating a connection I went back to import a solution again ( Audit Logs).  You can see below I clicked on the select connection and it did not show me. 
![image](https://user-images.githubusercontent.com/59940632/93908045-20ada400-fccc-11ea-851c-3bfc4d623af2.png)

When I click on the new connection I get the below error message. 
![image](https://user-images.githubusercontent.com/59940632/93907828-d88e8180-fccb-11ea-822f-f346ee23b6ef.png)

I have the custom connector when I go to Data -> Custom Connector 
![image](https://user-images.githubusercontent.com/59940632/93908244-69655d00-fccc-11ea-918f-6d1e746c4c09.png)

**Comment by manuelap-msft** [comment]
No, that's correct - the flow and solution are using a new feature called Connection References (read more about it here: https://flow.microsoft.com/en-us/blog/move-flows-across-environments-without-resetting-connections/) - During the import, you specify the connection to use for the connector, which is stored in the connection reference object. admin_Office365AuditCustomCoEConnection is the name of the connection reference, not the connector. 

If you select Edit on the Custom Connector in the Core Solution and navigate to the Test tab, does it show your connection there?
![image](https://user-images.githubusercontent.com/55705669/93910106-8adc3e00-fcf8-11ea-943d-20a740431f9f.png)


And if not, can you create it from there first?

**Comment by aakashpatel660** [comment]
yes, I am able to see the connection when I edit the custom connector.

![image](https://user-images.githubusercontent.com/59940632/93910695-76378000-fccf-11ea-99a9-6eb0167fe093.png)

**Comment by manuelap-msft** [comment]
Hello,

we had to revert the Audit Log solution back to the old version that didn't use Connection References due to a product bug that meant custom connectors and connection references didn't work. The version using Connection References has been removed from our download, and customers can use the old version still. 

We will update the CoE Starter Kit Audit Log solution as soon as custom connectors and connection references work.

Thanks for the feedback and using the CoE Starter Kit
